sdk/src/test/AllTests.ls

```diff
@@ -30,9 +30,11 @@ package {
      */
     public class AllTests {   
         public static function main() {
-            var result:TestResult = TestRunner.runAll((AllTests as Type).getAssembly());
-            
-            Process.exit(result.typeReport.successful ? 0 : 1);
+            TestRunner.onComplete += function(result:TestResult) {
+                
+                Process.exit(result.typeReport.successful ? 0 : 1);
+            };
+            TestRunner.runAll((AllTests as Type).getAssembly(), true);
```

Comment:
Let's not queue. Let's set a flag while tests are running and error if you try to run two sets. A way to query if tests are running is also reasonable.